lexos/interfaces/base_interface.py

```diff
@@ -14,12 +13,13 @@
 base_view = Blueprint('base', __name__)
 
 
-def detect_active_docs():
-    """
-    This function (which should probably be moved to file_manager.py) detects
-    the number of active documents and can be called at the beginning of each
-    tool.
+def detect_active_docs() -> int:
+    """detects the number of active documents. This function can be called at
+    the beginning of each tool.
```

Comment:
You cannot have a change of line in the one line summary.

Please re-read the documentation standards